### PR TITLE
WIP: Use Biggy search engine

### PR DESCRIPTION
**Pull request by hiagolcm**
#### What problem is this solving?

Change from SOLR to Biggy search engine

#### How should this be manually tested?

[Workspace](https://searchresolverv1--storecomponents.myvtex.com/)

#### Checklist/Reminders

- [x] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: node/commons/products.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      products.push(convertBiggyProduct(product, segment && segment.channel))
+    } catch (e) {
+      // TODO: add logging
+    }
+  })
+
+  return products
+}
+
+export const productsVtex = async (searchResult: any, ctx: any) => {
```
Comment (chrsmutti) [inline node/commons/products.ts:20]: Changing `VTEX` prefixes and suffixes into `Catalog` or `Portal` would be more representative.

File: node/resolvers/search/index.ts [hunk 2]
```diff
@@ -243,60 +184,33 @@ export const queries = {
...
+            ? `${attributePath}/${facet.key}/${facet.value}`
+            : attributePath,
+        ''
+      ),
+      query: args.fullText,
+      tradePolicy: segment && segment.channel,
     }
 
-    args.map = args.map && decodeURIComponent(args.map)
-    const translatedQuery = await translateToStoreDefaultLanguage(ctx, query!)
```
Comment (chrsmutti) [inline node/resolvers/search/index.ts:258]: Maybe we should keep the translation functionality for multi-language stores? 🤔

Comment (hiagolcm) [reply node/resolvers/search/index.ts:258]: done!

File: node/commons/products.ts (hunk 1 above)
Comment (hiagolcm) [reply node/commons/products.ts]: done!

File: node/clients/biggy-search.ts [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+        query,
+        page,
+        count,
+        sort,
+        operator,
+        fuzzy,
+        bgy_leap: leap ? true : undefined,
+      },
+      metric: 'search-result',
+      validateStatus: (status: number) => {
```
Comment (chrsmutti) [inline node/clients/biggy-search.ts:106]: The split APIs do not use `302` to denote redirects, you can remove this validation :)

**Pull request closed without merging**